fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -1111,7 +1107,8 @@ public SingleResourceResult<? extends Resource> doRead(String type, String id, b
      * @return a {@link SingleResourceResult} containing the Resource
      * @throws Exception
      */
-    private SingleResourceResult<? extends Resource> doRead(String type, String id, boolean throwExcOnNull, boolean includeDeleted,
+    // TODO remove contextResource?  it doesn't seem to ever be populated...
```

Comment:
@punktilious do you happen to know what this "contextResource" parameter is for?  
We use it when we construct the persistence event for doRead (line 1146 below), but I havn't found any callers that set it to anything other than null...

Reply:
it seems to be a remnant for our DSTU2 server...I see it going back more than 2 years...